client/my-sites/plans/jetpack-plans/product-lightbox/quantity-dropdown.tsx

```diff
@@ -20,7 +20,8 @@ interface QuantityDropdownProps {
 const QuantityDropdown: FC< QuantityDropdownProps > = ( { product, siteId, onChangeProduct } ) => {
 	const dispatch = useDispatch();
 	const listPrices = useItemPrice( siteId, product, product?.monthlyProductSlug || '' );
-	const isProductWithTierList = isJetpackAISlug( product.productSlug );
+	const isProductWithTierList =
+		isJetpackAISlug( product.productSlug ) || isJetpackStatsSlug( product.productSlug );
```

Comment:
Would see it as a constant, so it can be easier to control in the future. What do you think?

Reply:
That's a good idea. I went through different ways to do this but for some reason that didn't cross my mind 😅